### PR TITLE
Add chapters description

### PR DESCRIPTION
**Pull request by meaboutsoftware**
THis PR contains the overview and short (summary) description of each chapter that we have (will have) in our repository.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -164,20 +164,72 @@ There is no business logic and there is also no need to have an object represent
...
 
 #### Summary
+
 After deeper thinking about our subdomains, we decided for following architectural patterns that will be applied in one of 4 chapters:
 
 image::Assets/subdomains_architectural_patterns.jpg[]
 
 ## Chapters
 
+In the beginning of every greenfield projects we need to make a lot of decisions. Quite often we are biased by confrences, meetups, friends and colleagues. As a result we decide for too complex architecture. This means starting with:
```
Comment (kamilbaczek) [inline README.adoc:174]: First sentence is about so called "Software Project Paradox" https://beyond-agility.com/project-paradox/. I like visualization like on this blog post. We can create quickly something similar. This is just suggestion. We can skip it as well if you think current version is good enough :D

File: README.adoc [hunk 2]
```diff
@@ -164,20 +164,72 @@ There is no business logic and there is also no need to have an object represent
...
+Other problem is related to selecting too trivial architecture for too long (this happens less times than "overcomplicated" one). This means that we just add code to a monolith, new features flood our codebase and then it becomes a big ball of mud.
+
+What we would like to present you in our story is the evolutionary approach that will tackle most of the applications that you work with.
+
 ### Chapter 1: Initial Architecture
 
-### Chapter 2: Projects Separation
+In this chapter we will show you how to start your solution architecture. We start with modularisation from day one of the application, but modules are only separated by namespaces (there is only one project for the production code called ```Fitnet```). Each process that occurs in each module is vertically sliced - all the code is covered in each process namespace. This way we gain 2 advantages:
+
+- All the code for each process is in just one namespace, so there is no need to look around in folders like ```Controllers, Entities, Commands, Queries``` etc. Everything is just in e.g. ```SignContract```
```
Comment (kamilbaczek) [inline README.adoc:192]: I think one of the biggest advantage is productivity in this approach. When you starting designing, you are not distracted by creating namespaces, renaming. 

We could add something like that "- developer is not distructed by creating project and managing references. Is focused only on design crucial business process and validate ideas."

What yo think?

Comment (meaboutsoftware) [reply README.adoc:192]: Done

File: README.adoc (hunk 1 above)
Comment (meaboutsoftware) [reply README.adoc:174]: Done

**Review by kamilbaczek — APPROVED** [review]
Good work 👍

**Pull request merged**